### PR TITLE
Change generated Triton kernel prefix to 'triton_kernel' to make it more distinguishable

### PR DESCRIPTION
**Pull request by yf225**
Current prefix is `kernel_` which is too generic to assert that it exists in the profiler trace. Changing this prefix to something more distinguishable.

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire @jansel

### CODE REVIEW

**Review by ngimel — APPROVED** [review]

**Comment by ngimel** [comment]
Ugh, so since this name is used in 2 places, maybe we should put it into config (or otherwise make sure that these 2 use sites don't go out of sync).

File: torch/_inductor/codegen/triton.py [hunk 1]
```diff
@@ -1277,7 +1277,9 @@ def codegen_node_schedule(self, node_schedule, numel, reduction_numel):
         else:
             kernel_name = wrapper.next_kernel_name()
             wrapper.kernels[src_code] = kernel_name
-            subs_name = kernel_name if config.triton.ordered_kernel_names else "kernel"
+            subs_name = (
+                kernel_name if config.triton.ordered_kernel_names else "triton__kernel"
```
Comment (jansel) [inline torch/_inductor/codegen/triton.py:1281]: I'd suggest just `triton_kernel` rather than double `_`

**Review by ngimel — APPROVED** [review]

**Comment by yf225** [comment]
We can close this PR if https://github.com/pytorch/pytorch/pull/88624 is merged.

**Pull request closed without merging**